steward/src/commands/simulate.rs

```diff
@@ -0,0 +1,76 @@
+//! `Cosmos mode signer` subcommand - example of how to write a subcommand
+
+/// App-local prelude includes `app_reader()`/`app_writer()`/`app_config()`
+/// accessors along with logging macros. Customize as you see fit.
+use crate::{
+    application::APP,
+    prelude::*,
+    server::{self, with_tls},
+    simulate::SimulateHandler,
+    tenderly::validate_tenderly_config,
+};
+use abscissa_core::{clap::Parser, Command, Runnable};
+use steward_proto::steward::simulate_contract_call_server::SimulateContractCallServer;
+
+/// Runs the Simulate server which uses Tenderly to simulate contract calls
+#[derive(Command, Debug, Default, Parser)]
+#[clap(
+    long_about = "Simulates contract calls using Tenderly. Tenderly credentials must be set in the config file to use."
+)]
+pub struct SimulateCmd {
+    /// Run the gRPC server with TLS enabled
+    #[clap(long)]
+    pub use_tls: bool,
+}
+
+impl Runnable for SimulateCmd {
+    /// Simulate the application.
+    fn run(&self) {
+        let config = APP.config();
+
+        info!("Starting application");
+        abscissa_tokio::run(&APP, async {
+            validate_tenderly_config(&config);
+
+            // Reflection required for certain clients to function... such as grpcurl
```

Comment:
This isn't required for `grpcurl` to function, but it is required to use the `list` or `describe` commands with `grpcurl`.